### ISSUE TITLE
Empty lines are removed when an accented character is encountered

### ISSUE THREAD

**Issue by NiavlysB** [bhargavrpatel/gulp-prettier#7]
The formatting works great until the point where there is an accented character in the file. All the remaining lines are then reformatted with all the empty lines stripped out.

### Steps to reproduce
Create `test.js`:
```
let a = 1;

// The blank line above is preserved
let b = 2;

// Now, accented characters
let c = [
"Des caractères accentués qui cassent tout!" ] ;

// The blank line above is not preserved
let d = 0;
```

And a gulpfile with this very basic task:
```
gulp.task('test', () => {
   var a = gulp.src("test.js")
      .pipe(prettier())
      .pipe(gulp.dest('test'));
});
```

The end result is as follows:
```
let a = 1;

// The blank line above is preserved
let b = 2;

// Now, accented characters
let c = [
  "Des caractères accentués qui cassent tout!"
]; // The blank line above is not preserved
let d = 0;
```

### Expected result
The formatted file should be:
```
let a = 1;

// The blank line above is preserved
let b = 2;

// Now, accented characters
let c = ["Des caractères accentués qui cassent tout!"];

// The blank line above is not preserved
let d = 0;
```

This is not an issue with prettier, since calling it directly works fine (`node node_modules/prettier/bin-prettier.js test.js`).
Using `gulp-prettier-plugin` seems to works fine as well, so I'm going to use that one for now, but I thought I should report this.

**Comment by TheDancingCode** [comment]
Closed by updating prettier in 7d790fd1a41f24c084c4e43baa2460d2fe2295e3. A new version of gulp-prettier including this change will be published soon.

**Comment by TheDancingCode** [comment]
Now published in v2.0.0.
